src/components/round/results/ProjectsList.tsx

```diff
@@ -0,0 +1,112 @@
+"use client"
+import { Loader2 } from "lucide-react"
+import Image from "next/image"
+import React from "react"
+
+import ExternalLink from "@/components/ExternalLink"
+import ArrowLeftIcon from "@/components/icons/arrowLeftIcon"
+import { Button } from "@/components/ui/button"
+import { FundingRewardDetails } from "@/lib/types"
+
+interface Props {
+  projectRewards: FundingRewardDetails[]
+  loading: boolean
+  round: string | number
+  totalCount: number
+  handleLoadMore: () => void
+  isFetchingMore: boolean
+}
+
+const ProjectsList = ({
+  projectRewards,
+  loading,
+  round,
+  totalCount,
+  handleLoadMore,
+  isFetchingMore,
+}: Props) => {
+  const getProjectUrl = (projectName: string) => {
+    const formattedName = projectName.replace(/\s+/g, "-")
```

Comment:
hey @sbvhev could you change the name cleanup function to `replace(/ /g, '-').replace(/[^a-zA-Z0-9-]/g, '')`. This will match retropgfhub's url builder

Reply:
Fixed